text/0236-deprecation-ember-string.md

```diff
@@ -1,5 +1,9 @@
 ---
+Stage: Released
 Start Date: 2017-07-14
+Release Date: 2021-02-08
+Release Versions:
+  ember-source: v3.25.0
```

Comment:
This deprecation hasn't been fully released.

Reply:
@locks thanks! Didn't know that, removed!